Twig/GravatarExtension.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Ornicar\GravatarBundle\Twig;
 
+use Ornicar\GravatarBundle\Templating\Helper\GravatarHelper;
```

Comment:
We dont need this one, the property phpdoc should be removed or change to the interface aswell.